src/Platforms/AbstractPlatform.php

```diff
@@ -4007,6 +4044,14 @@ public function supportsColumnLengthIndexes(): bool
         return false;
     }
 
+    /**
+     * Whether the platform supports variable precision time.
+     */
+    public function supportsVariablePrecisionTime(): bool
```

Comment:
After having deprecated a dozen of `supports*()`, `prefers*()` and `uses*()` platform methods, do we want to introduce a new one?

Reply:
Patch set reworked without `supports()` method. Variable precision time/datetime is implemented on all platforms with the following exceptions:

- Sqlite - no support because Sqlite does not differentiate between e.g. DATETIME(6) and DATETIME and stores dates as strings.
- DB2. Support only for datetime not time.
- Oracle. Support only for datetime not time.

Immutable/mutable and with/without timezone all implemented except as above.